config/packages/security.yaml

```diff
@@ -22,8 +22,8 @@ security:
     access_control:
         - { path: '^/manage/admin', role: 'ROLE_ADMIN' }
         - { path: '^/manage', role: 'ROLE_USER' }
-        - { path: '^/connect/(mellon)$', role: 'IS_AUTHENTICATED_ANONYMOUSLY' }
-        - { path: '^/connect/(mellon)/check', role: 'ROLE_USER' }
+        - { path: '^/connect/(mellon|inacademia)$', role: 'IS_AUTHENTICATED_ANONYMOUSLY' }
+        - { path: '^/connect/(mellon|inacademia)/', role: 'ROLE_USER' }
```

Comment:
So it's only specifically usable with inacademia and not with a generic OpenID Connect provider?

Reply:
No, the point is I asked for this return url to @surfnet-niels because I thought it is more semantic for our use case. Now, I think I was wrong and this should be 'oidc'. This wil require another change in the return url for our demo.

Reply:
Changed to oidc